components/analytics/docker-compose.analytics.yml

```diff
@@ -63,7 +63,7 @@ services:
       DJANGO_LOG_SERVER_PORT: 5000
       DJANGO_LOG_VIEWER_HOST: kibana
       DJANGO_LOG_VIEWER_PORT: 5601
-      no_proxy: kibana,logstash,${no_proxy}
+      no_proxy: kibana,logstash,nuclio,${no_proxy}
```

Comment:
why do we need it here?

Reply:
Without this change, if we create images with the analytic component, no_proxy will be overridden by this value without `nuclio`.

Reply:
OK. I see. I thought the change was necessary for analytics itself.